ios/RNModelRepository.mm

```diff
@@ -0,0 +1,39 @@
+
+#import "RNModelRepository.h"
+#import "RNPixelBufferUtilities.h"
+
+#import "TIOModelRepository.h"
+#import "TIOModelBundle.h"
+#import "TIOModelUpdater.h"
+
+@interface RNModelRepository()
+
+@property TIOModelRepository* repository;
+
+@end
+
+@implementation RNModelRepository
+
+RCT_EXPORT_MODULE();
+
+RCT_EXPORT_METHOD(initialize:(NSString*)baseUrl) {
+    NSURL *URL = [NSURL URLWithString:baseUrl];
+    self.repository = [[TIOModelRepository alloc] initWithBaseURL:URL session:nil];
+}
+
+RCT_EXPORT_METHOD(updateModel:(NSString*)pathToFile callback:(RCTResponseSenderBlock)callback) {
+    //Get model bundle from url
+    TIOModelBundle *bundle = [[TIOModelBundle alloc] initWithPath:pathToFile];
+
+    //Use updator to actually update model
+    TIOModelUpdater *updater = [[TIOModelUpdater alloc] initWithModelBundle:bundle repository:self.repository];
+    [updater updateWithValidator:nil callback:^(BOOL updated, NSURL * _Nullable updatedBundleURL, NSError * _Nullable error) {
+        NSMutableDictionary *result = [[NSMutableDictionary alloc] init];
+        [result setValue:@(updated) forKey:@"updated"];
+        [result setValue:updatedBundleURL forKey:@"updatedBundleURL"];
+        return callback(@[error, result]);
```

Comment:
Also note that you don't need to return the value of the callback. That gnarly: 

```objc
clallback:^(BOOL updated, NSURL * _Nullable updatedBundleURL, NSError * _Nullable error) {

}
```

updater callback block that you've got your code in works like a javascript function callback so returning from it doesn't do anything.

Reply:
Just a habit from the async JS world lol. It prevents any subsequent, accidental callback invocations. Anyways, I removed the return.